### PR TITLE
675 move ci windows installer

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -95,7 +98,39 @@ jobs:
...
           retention-days: 7
+
+  build-packages-windows:
+    name: Build .msi package for amd64
+    needs: build
+    defaults:
+      run:
+        working-directory: ./installer
+
+    runs-on: windows-latest
```
Comment (Thykof) [inline .github/workflows/build.yml:113]: `runs-on: windows-latest` it's not a good practice to rely on latest versions (same for macos). Is there a specific reason?

File: installer/windows/build_installer.py [hunk 2]
```diff
@@ -64,6 +81,10 @@ def build_massastation():
         ],
         check=True,
     )
+    os.rename(
+        os.path.join("..", "cmd", "massastation", "MassaStation.exe"),
```
Comment (Thykof) [inline installer/windows/build_installer.py:85]: here why it's `MassaStation.exe` and not var `MASSASTATION_BINARY`?

**Review by Thykof — CHANGES_REQUESTED** [review]

File: installer/windows/build_installer.py (hunk 2 above)
Comment (thomas-senechal) [reply installer/windows/build_installer.py:85]: When we run `fyne package`, the `-name` correspond to the name of the package we want to make. In the case of Windows, the resulting package is a `.exe` named as the `-name` option, so in that case, `MassaStation.exe`. That's why we need to move it, and rename it.

File: .github/workflows/build.yml (hunk 1 above)
Comment (thomas-senechal) [reply .github/workflows/build.yml:113]: There is no real reason for that.
Do you think I should use a specific version ? Or is the latest windows version fine (I'm talking about windows-2022) ?

File: installer/windows/build_installer.py (hunk 2 above)
Comment (Thykof) [reply installer/windows/build_installer.py:85]: okay, you can add a comment in the code explaining that.

File: .github/workflows/build.yml (hunk 1 above)
Comment (Thykof) [reply .github/workflows/build.yml:113]: @peterjah WDYT?

File: installer/windows/build_installer.py (hunk 2 above)
Comment (thomas-senechal) [reply installer/windows/build_installer.py:88]: Done

**Review by Thykof — APPROVED** [review]
LGTM

File: .github/workflows/build.yml (hunk 1 above)
Comment (gregLibert) [reply .github/workflows/build.yml:113]: No, we should use lastest.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -95,7 +98,39 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v3
```
Comment (gregLibert) [inline .github/workflows/build.yml:115]: Set a name to this step.

**Review by gregLibert — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**